### ISSUE TITLE
45410813 - Skate.

### ISSUE THREAD

**Issue by nfxw** [xenia-canary/game-compatibility#319]
### Validation

- [X] I've read the [FAQ](https://github.com/xenia-canary/xenia-canary/wiki/FAQ).
- [X] The Xenia build used is from a Canary branch. (not UWP/MLBS/AlexVS/master/pull requests, etc)
- [X] This issue isn't for help (tech support) with Xenia, or a game.
- [X] I've checked if this game doesn't already have a compatibility report.

### Wiki page (optional)

https://github.com/xenia-canary/game-compatibility/wiki/...

### Xenia Canary version

https://github.com/xenia-canary/xenia-canary/commit/c71649161ee7127d3e05bc2fba57f49173f65fcf

### Issues

Fully playable.
720p - no issues.
1440p - minor flickering in some cutscenes, no flickering in gameplay.

### Log

[xenia.log](https://github.com/user-attachments/files/17981578/xenia.log)

### Screenshots

![1](https://github.com/user-attachments/assets/b474d906-149f-420c-a15c-d8450ef1102f)

### Labels

state-playable

**Comment by nfxw** [comment]
Finished the game, fully playable from start to finish.
![1](https://github.com/user-attachments/assets/ae2041d3-88bd-4e52-8281-e6cd214f862a)

**Comment by CognizantDoor69** [comment]
hey guys, this game keeps freezing on Xenia Canary. Is this a joke? You said that skate is playable, but on my computer and on the emulator, skate freezes every time i start the gameplay. Do you have any fixes for this?

**Comment by A1eNaz** [comment]
> hey guys, this game keeps freezing on Xenia Canary. Is this a joke? You said that skate is playable, but on my computer and on the emulator, skate freezes every time i start the gameplay. Do you have any fixes for this?

works on latest canary with no config changes
![Image](https://github.com/user-attachments/assets/5bd280ae-3bd0-4794-b6bf-59f46e2c6a84)

**Comment by kortul** [comment]
The 360/ Xenia version seems to be beter off at the moment than the PS3/ RPCS3 version. On RPCS3 there are occasional desync audio issues during cutscenes, sometimes sound effects delay occur and the career photographs do not appear in the magazines.

**Comment by Shell94845** [comment]
yep its not playable at all i dont know what uv tested on, i got 4090, i9 14k and i got the exact same issue as the guy above it never worked despite trying all settings, when i do vulkan it work but the screen is only a blurry mess.  On steam deck it work flawlessy though. Also it seems display resolution is broken and will adapt to your display

**Comment by Shell94845** [comment]
The sound stop and 1 second after the game freeze i guess this got to do with d3d12 since vulkan and steam deck work

**Comment by Shell94845** [comment]
Also optimised settings doesnt work at all

**Comment by Shell94845** [comment]
<img width="1284" height="752" alt="Image" src="https://github.com/user-attachments/assets/58d6a85c-be30-4e0d-8e8c-8d56d3110be3" /> the vulkan joy

**Comment by Shell94845** [comment]
i was going to write that it finaly worked with readback resolve to full , windowed mode but when i was about to type i crashed :D :D :D

**Comment by goldislead** [comment]
@Shell94845 

Please avoid spamming threads with single sentence posts; this is not a Discord chat.

**Comment by nfxw** [comment]
@Shell94845 
To fix audio freeze, try to change use_dedicated_xma_thread to false and xma_decoder to "old".
Also Vulkan is heavily incomplete in xenia canary, so stay with d3d12 then.

**Comment by ehjr5u** [comment]
I would say while playing this game, it likes to randomly freeze from a few frames, to a completely frozen screen that lasts from multiple seconds, or doesn't load. although I can sometimes control the skater.

**Comment by Shell94845** [comment]
> [@Shell94845](https://github.com/Shell94845) To fix audio freeze, try to change use_dedicated_xma_thread to false and xma_decoder to "old". Also Vulkan is heavily incomplete in xenia canary, so stay with d3d12 then.

i tried but still freezed (but i played a little longer before freeze) audio freeze always lead to full freeze a second after

**Comment by nfxw** [comment]
> > [@Shell94845](https://github.com/Shell94845) To fix audio freeze, try to change use_dedicated_xma_thread to false and xma_decoder to "old". Also Vulkan is heavily incomplete in xenia canary, so stay with d3d12 then.
> 
> i tried but still freezed (but i played a little longer before freeze) audio freeze always lead to full freeze a second after

Tried on 14c2814, no freezes, dedicated_xma_thread on false, decoder is old.
Also i'm using the 0.0.0.1 version of the game, usa.

<img width="1920" height="1200" alt="Image" src="https://github.com/user-attachments/assets/9fa21eb8-a5ed-4803-a306-bf088bb2b710" />

**Comment by Shell94845** [comment]
okayyyy i did not updated and i was on xenia manager, works flawlessy now sorry

**Comment by nfxw** [comment]
Completed the game to 100%.

<img width="1920" height="1200" alt="Image" src="https://github.com/user-attachments/assets/6c84ae85-f977-4597-8e80-e1014b86580c" />
<img width="1920" height="1200" alt="Image" src="https://github.com/user-attachments/assets/c31032f2-0edf-4287-ba85-93296ebbd98c" />
